server/api/v1/controllers/quotesApiController.js

```diff
@@ -0,0 +1,279 @@
+"use strict";
+
+const quotesManager = require("../../../../backend/quotes/quotes-manager");
+const moment = require("moment");
+
+exports.getQuotes = async function(req, res) {
+    const quotes = await quotesManager.getAllQuotes();
+
+    const formattedQuotes = quotes.map(q => {
+        return formatQuote(q);
+    });
+
+    res.json(formattedQuotes);
+};
+
+exports.getQuote = async function(req, res) {
+    let { quoteId } = req.params;
+
+    if (quoteId == null) {
+        return res.status(400).send({
+            status: "error",
+            message: "No quoteId provided"
+        });
+    }
+
+    quoteId = parseInt(quoteId);
+    if (isNaN(quoteId)) {
+        return res.status(400).send({
+            status: "error",
+            message: "Invalid quoteId provided"
+        });
+    }
+
+    const quote = await quotesManager.getQuote(quoteId);
+
+    if (quote == null) {
+        return res.status(404).send({
+            status: "error",
+            message: "Quote " + quoteId + " not found"
```

Comment:
Although a lot of our legacy code is bad about this, for new stuff we generally want to always use string template literals instead of string concatenation. IE:
```js
{
  //...
  message: `Quote ${quoteId} not found`
}
```

Looks like there's a few places here that an be updated to template literals.

Reply:
Fixed multiple